app/api/csv/specs/importEntity.spec.js

```diff
@@ -0,0 +1,141 @@
+import path from 'path';
```

Comment:
@LaszloKecskes is this spec really necessary ? i mean this development is a performance improvement right, the "result" of everything should be the same, are we doing new things that were not done before ?, if this is the case do you think we can add those those tests to any existing spec that tests the import itself (the whole module) like csvLoader.spec ?

Reply:
Rewritten as discussed: to do the testing via CSVLoader.load, rather then directly the implementation. Still kept in a separate file, to not clutter the rest with unrelated checks.